paddle/fluid/operators/math/gru_compute.cu

```diff
@@ -30,10 +30,26 @@ struct GRUUnitFunctor<platform::CUDADeviceContext, T> {
     dim3 threads;
     dim3 grid;
     if (batch_size == 1) {
-      int frame_per_block = frame_size <= 1024 ? frame_size : 1024;
-      int frame_blocks = (frame_size + 1024 - 1) / 1024;
-      threads = dim3(frame_per_block, 1);
+      const int tiled_size = 16;
+      int frame_blocks = (frame_size * 2 + tiled_size - 1) / tiled_size;
+      threads = dim3(tiled_size, 1);
       grid = dim3(frame_blocks, 1);
+
+      detail::KeFastCollectiveGruGate<T,
```

Comment:
What is the meaning of `KeFastCollective`?

Reply:
means gpu kernel collect data into register